Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut

```diff
@@ -59,10 +59,15 @@ void function InfectionOnPlayerKilled( entity victim, entity attacker, var damag
 	if ( victim.GetTeam() == INFECTION_TEAM_SURVIVOR )
 		InfectPlayer( victim )
 
-	if ( attacker.IsPlayer() ) {
+	if ( attacker.IsPlayer() )
+	{
 		attacker.SetPlayerGameStat( PGS_ASSAULT_SCORE, attacker.GetPlayerGameStat( PGS_ASSAULT_SCORE ) + 1 )
 		SetRoundWinningKillReplayAttacker(attacker)
 	}
+	else
+	{
+		SetRoundWinningKillReplayAttacker(player) // everybody now gets to witness you dying to the game

```

Comment:
Variable `player` is not defined

Reply:
yeah thats just me being a moron and forgetting to change it when i moved my functions around to match most recent version, should be fixed now

Reply:
So, this is supposed to trigger when the player suicides to the map right? I tried testing this by dying in out of bounds areas and the final killcam didn't show up

Reply:
Yeah, not sure why that's not working for you, I'll test it on my computer in a sec, try and figure out where its fucking up

Reply:
seems to work fine for me? not sure why it's not working for you atm (most recent commit at least)

Reply:
Still can't trigger it. I tried killing myself by falling out of bounds or running out of 'out of bounds timer' while being the last survivor